### ISSUE TITLE
Improve error output for schema validation workflow

### ISSUE THREAD

**Issue by MoonlightKomorebi** [newrelic/newrelic-quickstarts#242]
## Summary
It was pointed in recent failures that the output produced when the validation workflow fails is not helpful. Here is a link to the failing workflow: [example](https://github.com/newrelic/newrelic-observability-packs/runs/3607638499), and the error output is: 

```none
 Error: packs/mlops/algorithmia/dashboards/algorithmia.json
	 must be string
	 must match a schema in anyOf
```

The output isn't always this unfriendly, it just depends on the specific error that is encountered. In general, we want to improve the output so that no matter the error the output generated points you to the error location, and a potential fix.

Before we start this work, we should identify the specific failure cases that we want to test for so that we can take a more rigorous approach this time around. 

## Acceptance Criteria
- [ ] TBD

**Comment by jpvajda** [comment]
@nr-kkenney do we still need to be concerned with this? I felt we did some improvement to error messages, but wasn't sure if we resolved this.

**Comment by nr-kkenney** [comment]
yes / no. i think we handled a couple specific cases (like [this](https://github.com/newrelic/newrelic-quickstarts/issues/557) one), but I imagine this could still crop up. Especially since we are going to switch how we validate.

But that can live in a new ticket.
